content/en/docs/develop/core-modules/upgrade.md

```diff
@@ -6,24 +6,124 @@ weight: 150
 {{< alert >}}
 **Note**
 
-XPLA Chain's upgrade module inherits from the Cosmos SDK's [`upgrade`](https://docs.cosmos.network/master/modules/upgrade/) module. This document is a stub and covers mainly important XPLA Chain-specific notes about how it is used.
+XPLA Chain's upgrade module inherits from the Cosmos SDK's [`upgrade`](https://docs.cosmos.network/v0.45/modules/upgrade/) module. This document is a stub and covers mainly important XPLA Chain-specific notes about how it is used.
 {{< /alert >}}
 
-The upgrade module allows live chains to be upgraded with breaking changes smoothly.
+`x/upgrade` is an implementation of a Cosmos SDK module that facilitates smoothly upgrading a live Cosmos chain to a new (breaking) software version. It accomplishes this by providing a `BeginBlocker` hook that prevents the blockchain state machine from proceeding once a pre-defined upgrade block height has been reached.
```

Comment:
remaining network name 👆

Reply:
and the network name should be capital letters, XPLA Chain

Reply:
https://github.com/xpladev/docs/pull/13/commits/5157a0daf4b64072bce960838115cf552c632aed